src/components/ArticleCard.jsx

```diff
@@ -23,25 +23,28 @@ const ArticleCard = () => {
       {isLoading ? (
         <p>Loading...</p>
       ) : (
-        <div className="card_article">
-          <h2 id="card_article_title">{article.title}</h2>
-          <h4 id="card_article_card_topic">
-            Topic: &nbsp;
-            <Link to={"/articles?topic=" + article.topic}>{article.topic}</Link>
-          </h4>
-          <h4 id="card_article_card_author">
-            Created by:&nbsp;
-            <Link to={"/articles?author=" + article.author}>
-              {article.author}
-            </Link>
-          </h4>
-          <img
-            id="card_article_card_img"
-            alt="article_cover"
-            src={article.article_img_url}
-          />
-          <p id="card_article_body">{article.body}</p>
-
+        <div>
+          <div className="card_article">
+            <h2 id="card_article_title">{article.title}</h2>
+            <h4 id="card_article_card_topic">
+              Topic: &nbsp;
+              <Link to={"/articles?topic=" + article.topic}>
```

Comment:
This appears to link to a url that doesn't currently work? Suspect this is for a future ticket. Please remove for now.

However you do need a way to get back to the home page from the individual review page, please include a way to do this.